cmd/nodecmd/create.go

```diff
@@ -88,6 +88,7 @@ will apply to all nodes in the cluster`,
 		RunE:         createNodes,
 	}
 	cmd.Flags().BoolVar(&useStaticIP, "use-static-ip", true, "attach static Public IP on cloud servers")
+	cmd.Flags().BoolVar(&separateMonitoringInstance, "separate-monitoring-instance", false, "host monitoring for all cloud servers on a separate instance")
```

Comment:
we need another flag to enable monitoring overall. As by default we install bunch of stuff like prometheus, grafana etc to the instances and "steal" resources avaliable for actual avalanchego